website_membership_gamification/wizard/gamification_badge_user_wizard.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2022 Manuel Regidor <manuel.regidor@sygel.es>
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record id="partner_view_badge_wizard_reward" model="ir.ui.view">
+        <field name="name">partner.gamification.badge.user.wizard.form</field>
+        <field name="model">gamification.badge.user.wizard</field>
+        <field name="arch" type="xml">
+            <form string="Reward with">
```

Comment:
Can't you inherit the view instead of overriding it? If you want a sperate form from the already existing you can also use `primary="1"`